vpr/src/place/move_utils.cpp

```diff
@@ -651,3 +651,16 @@ bool find_to_loc_uniform(t_logical_block_type_ptr type,
 
     return true;
 }
+
+std::string e_move_result_to_string(e_move_result move_outcome) {
```

Comment:
You don't need such a function. Just define an array/vector of strings that holds the names of your enum.
char* move_result_to_string[] = {"ACCEPTED", "REJECTED", "ABORTED"}

Then, you can convert the move result into string by addressing this array e.g. move_result_to_string[move_result]

Reply:
Duplicated comment. Vaughn already highlighted that.

Reply:
Agreed.  I still like a function wrapping the access to the array of strings though (so keep a utility function to turn move types into strings), but the indexing into an array is generally easier to read and probably a bit faster than a bunch of if statements.

Reply:
resolved